src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.php

```diff
@@ -317,5 +318,9 @@
             ->parent('notifier.transport_factory.abstract')
             ->tag('texter.transport_factory')
 
+        ->set('notifier.transport_factory.smsmode', SmsmodeTransportFactory::class)
+            ->parent('notifier.transport_factory.abstract')
+            ->tag('texter.transport_factory')
```

Comment:
Wait, this class is configured as `texter`, so it is supposed to support `SmsMessage`. `SmsMessage` has no possibility to add options...

Reply:
All good now after #48254